package.json

```diff
@@ -341,7 +341,7 @@
     "url-search-params-polyfill": "^8.1.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#0aea5769bad46b9da4e60f8d40c613ad7a1c6bad",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#d8a039099a64a198c45280bf4ca83cabd2a2968d",
```

Comment:
This hash will be updated to the one generated when [this pr](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/535) is merged into master.